Randomizer.py

```diff
@@ -1,24 +1,25 @@
 import random
 
 
-def randomly(list_of_students):
-    student_dict_reverted = {}
+def random_student(list_of_students):
+    student_dict = {}
     for elem in list_of_students:
-        student_dict_reverted[elem] = 0
-    print('Do you want to make your choice?', '\n', 'Press y to do it, n to exit')
+        student_dict[elem] = 0
+    # print('Do you want to make your choice?', '\n', 'Press y to do it, n to exit')
+    somestud = random.choice(list_of_students)
+    print(somestud)
+    student_dict[somestud] += 1
     while True:
-        choice2 = input()
-        if choice2 == 'n':
+        print('Proceed? (y/n)', end=' ')

```

Comment:
Forgot to mention this before, read documentation about `input()`, it takes some arguments that may alleviate you pain.